recipes/odysee/webview.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (Ferdium) => {
+  function getNotifications() {
+      let unreadNotifications = 0;
+      for (const notificationCounterElement of document.querySelectorAll(
+              ".notification__count"
+          )) {
+          unreadNotifications =
+              unreadNotifications +
+              Ferdium.safeParseInt(notificationCounterElement.textContent);
+      }
+
+      unreadNotifications = unreadNotifications / 1;
```

Comment:
What's the point of this division by 1?

Reply:
I apologize about that. I have removed it.